package.json

```
@@ -395,7 +395,7 @@
     "prop-types": "^15.6.1",
     "punycode": "^2.1.1",
     "qrcode-generator": "1.4.1",
-    "qrcode.react": "^1.0.1",
+    "qrcode.react": "3.1.0",
```

We use `^` ranges for most dependencies, to make updates easier.

Is there a specific reason that we should pin this to 3.1.0?

```suggestion
    "qrcode.react": "^3.1.0",
```

---

@Gudahtt We used "3.1.0" is because we did our internal testing with that specific version.

---

Thank you for your swift response @Gudahtt . 
Our proposal for pinning to version 3.1.0 stems from our extensive internal testing using 3.1.0.
We've ensured the expected QR codes for URs are generated correctly. Allowing automatic updates for versions >= 3.1.0 < 4.0.0 may introduce changes that might not have been tested in our environment, leading to potential issues down the road.
By specifying 3.1.0, the goal is to maintain control over when and how this dependency is updated. Any future updates to newer versions can be planned and tested appropriately.

---

Got it. Could you update this to `^3.1.0` please? The lock file will ensure that v3.1.0 remains the specific version used.